### DISCUSSION TITLE
BitDepth with Channel Specified as CMYK for RGB  image returns value

### DISCUSSION THREAD

**Discussion by manjunthcm** [dlemstra/Magick.NET#624] [General] [answered]
hi
i have the following pieces of code
int depthofColor = tiffImage.BitDepth(Channels.CMYK);
and
int depthofClr = tiffImage.BitDepth(Channels.Cyan);

when my tiffimage is of RGB color Space, why would the above two lines of code return 8 even though i have Specified channels for BitDepth ?

**Comment by dlemstra** [comment, marked as answer]
This is happening because `Cyan=Red`, `Magenta=Green` and `Yellow=Blue` and this methods will return the minimum depth for the specified channels. So in your case you are asking the bit depth for the Red, Green and Blue channels in the first call and only for the Red in the second call.
